src/index.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+import { BrowserRouter as Router } from 'react-router-dom';
 import reportWebVitals from './reportWebVitals';
-
+import './index.css';
 import App from './App';
 // import Calc from './components/Calc';
```

Comment:
### 🛑 Required Change

- [ ] Kindly delete commented out code to maintain clean readable code

Reply:
Done, thank you :-)